mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -322,7 +333,7 @@ abstract class RecommendationTests(sc: SparkContext) extends PerfTest {
     val implicitRatings: Boolean = booleanOptionValue(IMPLICIT)
 
     val data = DataGenerator.generateRatings(sc, numUsers, numProducts,
-      math.ceil(numRatings * 1.25).toLong, implicitRatings,numPartitions,seed)
+      math.ceil(numRatings).toLong, implicitRatings,numPartitions,seed)
```

Comment:
We use `1.25` so after 80/20 split we have roughly the requested number of ratings. If we still want to compare with 1.4, maybe it is not worth changing it. Otherwise, it should be okay.